TYPO3.Eel/Classes/TYPO3/Eel/Helper/StringHelper.php

```diff
@@ -555,4 +574,5 @@ public function allowsCallOfMethod($methodName)
     {
         return true;
     }
+
```

Comment:
This addition of a line break is the reason for the Style-CI error, can easily be fixed ;)